### DISCUSSION TITLE
Access note attributes via js

### DISCUSSION THREAD

**Discussion by twicemice** [TriliumNext/Trilium#3343] [Q&A] [answered]
Hi again
I would like to know how to access with JS to the note attributes "created" and "modified". 
Is interesting for me also, access the INFO field from the notes created with the web clipper. 
Thanks for your attention.

**Comment by zadam** [comment, marked as answer]
e.g. (type: JS fronted / JS backend)

```javascript
console.log(api.startNote.dateCreated); // also dateModified, utcDateCreated, utcDateModified
```

See also https://github.com/zadam/trilium/wiki/Script-API

https://github.com/zadam/trilium/wiki/Scripts

**Reply by twicemice** [reply]
Thank you very much. 
Sorry I haven´t found "dateCreated" or "dateModified" in the API.
Let's keep learning!

**Reply by zadam** [reply]
Note properties are here: https://zadam.github.io/trilium/backend_api/Note.html
